docs/coding-guidelines.md

```diff
@@ -332,6 +332,22 @@ do not decrease the overall code coverage of the codebase - this is aligned with
 goal to increase coverage over time. Keep track of execution time for your unit
 tests and try to keep them as short as possible.
 
+## Semantic Conventions compatibility
+
+When adding new metrics, attributes or entity attributes to a Collector's component (receiver, processor etc), the
+[Semantic Conventions](https://github.com/open-telemetry/semantic-conventions) project should be checked first
+to see if those are already defined as Semantic Conventions.
+It's also important to check for any open issues that may already propose these or similar Semantic Conventions.
+If no such Semantic Conventions are defined in the Semantic Conventions project, the component’s code owners
+should consider initiating that process first.
```

Comment:
Can you link to the process (if there is one documented)?

Reply:
Maybe just https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md?

Reply:
Makes sense! Added: [c5aece7](https://github.com/open-telemetry/opentelemetry-collector/pull/13101/commits/c5aece73af8a7920a0131c6bb7aa2331d504573b)